pkg/terraform/gather/ovirt/ip.go

```diff
@@ -69,52 +76,54 @@ func findVirtualMachineIP(c *ovirtsdk4.Connection, moRefValue string) (string, e
 				ipres, hasAddress := ip.Address()
 				if hasAddress {
 					if checkPortIsOpen(ipres, bootstrapSSHPort) {
-						logrus.Debugf("ovirt vm id: %s , found usable bootstrap IP %s", moRefValue, ipres)
+						logrus.Debugf("ovirt vm id: %s , found usable  IP Address: %s", moRefValue, ipres)
```

Comment:
Is this log useful to the user anyhow or is this just debug code?

Reply:
its for debug